tools/db_stress.cc

```diff
@@ -2834,17 +2834,8 @@ int main(int argc, char** argv) {
   ParseCommandLineFlags(&argc, &argv, true);
 #if !defined(NDEBUG) && !defined(OS_MACOSX) && !defined(OS_WIN) && \
   !defined(OS_SOLARIS) && !defined(OS_AIX)
-  rocksdb::SyncPoint::GetInstance()->SetCallBack(
```

Comment:
this code seemed weird (why run in direct I/O mode if it's not supported..), and it would've prevented using killpoint-only mode, so I deleted it